bodhi/server/static/js/search.js

```diff
@@ -65,7 +69,7 @@ $(document).ready(function() {
     $('#bloodhound .typeahead').typeahead({
         hint: true,
         highlight: true,
-        minLength: 2,
+        minLength: 3,
```

Comment:
There are 55 packages with names that are less than 3 characters (examples: ed and qt), and this change makes it a bit hard to find updates that match them since pressing enter doesn't sort the results that are an exact match to the top like the search box does. I haven't observed performance problems with the setting as is, have you?

Reply:
Oh, I haven't considered those packages... I'm going to revert this change.

The performance problem I see is in the time needed to search for updates (and to a lesser extent in the search for overrides). But that's because the query is made looking for `%queryterm%`, which with few characters returns a huge amount of results (for example, with 'ed' typed in the search bar, we get nearly every update in the database, since it matches 'FedORA-' update aliases.